cl/cltypes/solid/attestation.go

```diff
@@ -49,6 +50,7 @@ func NewAttestionFromParameters(
 	signature [96]byte,
 ) *Attestation {
 	a := &Attestation{}
+	binary.LittleEndian.PutUint32(a.staticBuffer[:4], aggregationBitsOffset)
```

Comment:
ah where is the doc regarding the bytes content of attestation

Reply:
it is the ssz encoding: https://www.ssz.dev/